include/Uhr.h

```diff
@@ -158,6 +158,7 @@ const long interval = 1000; // 1 Sekunde
 uint32_t uhrzeit;
 uint16_t Word_array[MAX_ARRAY_SIZE] = {0};
 uint16_t Word_array_old[MAX_ARRAY_SIZE] = {0};
+bool Changes_in_Parameter = false;
```

Comment:
evtl. umbenennen in `parameters_changed`?

Reply:
Oh man, meine Schreibweisen. Ich war wahrscheinlich echt im Tunnelblick. 

Hier würde ich allerdings auf den Kommentar darüber verweisen.